File: JavaScript/Pattern1.js
// Pattern 1 
// * * 
// * * 

class Solution{
    Pattern1(N){
        for (let i = 0; i< N; i++){
            let row = "";
            for (let j=0; j<N; j++){
                row += "* ";
            }
            console.log(row);
        }

    }
}

const sol = new Solution();
const N = 2;
sol.Pattern1(N);

File: Python/Pattern1.py
# Pattern 1 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 
# * * * * * * * * * * 

class Solution:
    def pattern1(self, N):
        for i in range(N):
            for j in range(N):
                print("*", end=" ")
            print()



sol = Solution()
N = 10
sol.pattern1(N)

  